index.js

```diff
@@ -31,12 +31,6 @@ function HtmlWebpackPlugin(options) {
   if(this.options.template.indexOf('!') === -1) {
     this.options.template = require.resolve('./loader.js') + '!' + path.resolve(this.options.template);
   }
-  // Resolve template path
```

Comment:
Maybe I am wrong but I believe this is necessary if this plugin is used as a submodule and the template lies in the parent module (which is true for most use cases).